src/test/resources/it/pagopa/pn/cucumber/AvanzamentoNotificheWebhookB2b.feature

```diff
@@ -170,4 +170,28 @@ Feature: avanzamento notifiche webhook b2b
     When si creano i nuovi stream
     Then l'ultima creazione ha prodotto un errore con status code "409"
 
+  @clean @integrationTest
```

Comment:
Essendo il test collegato al funzionamento del mock eviterei di indicarlo come "@ integrationTest" essendo questa test suite destinata ad uso esterno, ed in più potrebbe essere il caso di marchiarlo come specifico per dev (@ dev) così da evitare la sua esecuzione in altri ambiente in cui non è garantito il corretto funzionamento.